src/chronify/store.py

```diff
@@ -159,6 +179,12 @@ def ingest_from_csv(
             conn.commit()
         self.update_table_schema()
 
+    def map_time(self, src_schema: TableSchema, dst_schema: TableSchema) -> None:
```

Comment:
Two versions of map time. I understand this one is a class method through store, would that be too confusing with the map_time() func?

Reply:
Users will see the Store method. They will not see the backend method. This is a forwarding situation.  I would argue that consistency is better for developers in the code.